tests/e2e/default/enketo/enketo-widgets.wdio-spec.js

```diff
@@ -83,6 +83,12 @@ describe('Enketo Widgets', () => {
     await loginPage.login(offlineUser);
   });
 
+  after(async () => {
+    await utils.deleteUsers([offlineUser]);
```

Comment:
Deleting users and calling `revertDb` is already part of the generic after defined here: https://github.com/medic/cht-core/blob/master/tests/wdio.conf.js#L336

I think you would only need to remove the form. Even though the generic revertdb should already cover that too.